### ISSUE TITLE
A default photo should be displayed whenever no photo URL is specified by participant.

### ISSUE THREAD

**Issue by zerefwayne** [opencodeiiita/Opencode-Collaborative-19#24]
![image](https://user-images.githubusercontent.com/28949397/51071304-dc53af00-1674-11e9-8159-381f34e4465e.png)

**Comment by thesmallstar** [comment]
@opencodebot claim

**Comment by Ace-Bansal** [comment]
Assigned to @thesmallstar

**Comment by thesmallstar** [comment]
@zerefwayne 
![screenshot_5](https://user-images.githubusercontent.com/42006277/51071756-3ad05b80-167c-11e9-9760-6feca39f70b4.png)

Archies11 provided the url, but did not upload the image. Does this also cover the following case( image not found) or only the blank url condition?

**Comment by zerefwayne** [comment]
Yes it does cover the blank url condition. But image not found needs to be managed as well.

**Comment by CoderJolly** [comment]
pls let me know, if th issue remains unsolved........i would like to try next

**Comment by zerefwayne** [comment]
@CoderJolly You can help @thesmallstar out with it. :+1: A healthy discussion always results in a good patch.

**Comment by thesmallstar** [comment]
@zerefwayne ok I will manage both the issues.

**Comment by thesmallstar** [comment]
@zerefwayne  issue was fixed and merged :)
